CareKit/CareKit/iOS/Utilities/OCKContactUtility.swift

```diff
@@ -55,7 +55,7 @@ struct OCKContactUtility {
         guard let components = components else { return nil }
         return nameFormatter.string(from: components)
     }
-
+/*
```

Comment:
It's okay to delete this instead of just commenting it out!

Reply:
Removed